tests/std/tests/VSO_0000000_type_traits/test.cpp

```diff
@@ -851,6 +858,13 @@ enum LLEnum : long long { xLongExample, yLongExample };
 enum class ExampleEnumClass { xExample, yExample };
 enum class LLEnumClass : long long { xLongExample, yLongExample };
 
+#if _HAS_CXX23
+STATIC_ASSERT(!is_scoped_enum<ExampleEnum>::value);

```

Comment:
This is mainly testing instantiation which I've just remembered is already covered in the other test. Should I remove this?

Also I kinda forgot about the test on line 80, should I just create some enums up top (or move these enums?) and move this test block?